data/sql/updates/pending_db_world/rev_1586023897838741800.sql

```diff
@@ -0,0 +1,42 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1586023897838741800');
+
+-- Delete queststarter for creature 20482
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10975;
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10976;
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10977;
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10981;
```

Comment:
Here you could do 
```sql
DELETE FROM `creature_queststarter` WHERE `id` = 20482 AND `quest` IN (10975, 10976, 10977, 10981);
```
To minimize the number of queries it has to run, which means it will run quicker in the DB.

Reply:
Done

Reply:
The day will come i finally remind that feature :sweat_smile:

Reply:
Haha 💃